app/src/main/res/values/strings.xml

```diff
@@ -559,10 +559,10 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_religion_for_wayside_shrine_title">What religion is represented at this shrine?</string>
     <string name="quest_parking_fee_title">Do you have to pay to park here?</string>
     <string name="quest_surface_value_unhewn_cobblestone">Unhewn cobblestone</string>
-    <string name="quest_parking_access_title">Is it restricted who may park here?</string>
-    <string name="quest_parking_access_private">It is private</string>
-    <string name="quest_parking_access_yes">It is open to the public (with or without fee)</string>
-    <string name="quest_parking_access_customers">It is only for customers</string>
+    <string name="quest_parking_access_title2">Who may park here? Parking may be free or paid.</string>
+    <string name="quest_access_yes">Open to general public</string>
+    <string name="quest_access_customers">Open only to people visiting a specific place (e.g. customers)</string>
+    <string name="quest_access_private">Not open to general public</string>
```

Comment:
```suggestion
    <string name="quest_access_private">Reserved for a specific person(s) (e.g. employees) </string>
```
Here's a start

Reply:
A bit long and awkward because it mixes singular and plural ("for a specific person(s)"). Also, it sounds it doesn't really fit to the classic "really" private parking on private property because "reserved" kind of implies that it is located on publicly accesible space but that parking in particular is reserved for xyz, e.g. a parking space being reserved for handicapped persons.

IMO "it is private" is better. But maybe "Reserved / Rented / Private" is fine too?

Reply:
Hm, good point about handicapped spots.

> private parking on private property

Is the parking quest asked for these?

Reply:
> Is the parking quest asked for these?

Wat? To find this out is the point of this quest 🤣

Reply:
I guess I haven't come across anything that is `access=private` besides parking lots marked "for employees only", so I wasn't sure if there was some additional filtering which happened to filter out driveways, etc.

Reply:
I guess in a city there's "parking along the street, by permit only"

Reply:
Closing this thread because I want to make modifications to multiple strings